python/mujinplanningclient/handeyecalibrationplanningclient.py

```diff
@@ -1,8 +1,13 @@
 # -*- coding: utf-8 -*-
-# Copyright (C) 2013-2015 MUJIN Inc.
-# Mujin controller client for bin picking task
+# Copyright (C) 2012-2023 Mujin, Inc.
+
+# system imports
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from typing import Any, Dict, List, Optional, Tuple, Union # noqa: F401 # used in type check
 
 # mujin imports
+from . import zmq
```

Comment:
```suggestion
if TYPE_CHECKING:
    from typing import Any, Dict, List, Optional, Tuple, Union # noqa: F401 # used in type check
    from . import zmq

# mujin imports
```
As in the other MR. Why are they separate, anyway?

Reply:
No particular reason. Just the all 4 clients would lead to pretty large MR, so I decided to split by client.